docs/details/shards/General/Hashed.md

```diff
@@ -1,6 +1,6 @@
 `(Hashed)` takes a single shard or a sequence of shards and hashes the output of the last shard (or the only shard) in this sequence. So this shard can be used to group together certain shards when you want their final output to be hashed.
 
-Like [(`Sub`)](../sub)), `(Hashed)` too uses `(->)` but unlike `(Sub)`, `(Hashed)` doesn't pass its input to all its shards in parallel. The input to `(Hashed)` becomes input to the first shard in the hashed sequence and the output of this shard becomes the input for the next shard in the hashed sequence (following the regular Shards data flow pattern).
+Like [(`Sub`)](../sub)), `(Hashed)` too uses `(->)` to group multiple shards. The input to `(Hashed)` becomes input to the first shard in the hashed sequence and the output of this shard becomes the input for the next shard in the hashed sequence (following the regular Shards data flow pattern).
```

Comment:
```suggestion
Like [(`Sub`)](../sub)), `(Hashed)` also uses `(->)` to group multiple shards. The input to `(Hashed)` becomes input to the first shard in the hashed sequence and the output of this shard becomes the input for the next shard in the hashed sequence (following the regular Shards data flow pattern).
```

Or just remove it. But "too" is usually for other uses.

Reply:
I simplified the sentence by removing the `Sub` reference altogether.
It was not really adding anything to the meaning.